src/services/doppler-api-client.ts

```diff
@@ -510,7 +510,7 @@ export class HttpDopplerApiClient implements DopplerApiClient {
 
       const { data } = await this.axios.request({
         method: 'GET',
-        url: `/accounts/${userAccount}/subscribers/${subscriberEmail}/permissions-history/${fieldName}`,
+        url: `/accounts/${userAccount}/lists/0/subscribers/${subscriberEmail}/permissions-history/${fieldName}`,
```

Comment:
Here it should be parametrized like this:
```suggestion
        url: `/accounts/${userAccount}/lists/{listId}/subscribers/${subscriberEmail}/permissions-history/${fieldName}`,
```

Reply:
No, no. In this case, `/lists/0` is a patch. It should not be there, but temporarily our backend is done in this way.

Reply:
Thanks for the clarification

Reply:
Finally this change was not necessary because it is already fixed in backend.